cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -223,6 +229,120 @@ public BodyConsumer deploy(HttpRequest request, HttpResponder responder,
   }
 
   /**
+   * Retrieves all adapters in a given namespace.
```

Comment:
Also, minor nitpick - Can you move all the ``/adapters`` APIs together either before or after the ``/apps`` APIs? Easier to discover.